docker/Dockerfile

```diff
@@ -49,79 +44,64 @@ RUN mvn -q --batch-mode -DskipTests -Dlicense.skip=true --also-make -pl zipkin-s
 # cache between builds.
 FROM openzipkin/java:${java_version} as zipkin-builder
 
-COPY --from=built /root/.m2 /root/.m2
-COPY --from=built /root/.npm /root/.npm
+COPY --from=install /root/.m2 /root/.m2
+COPY --from=install /root/.npm /root/.npm
 
 #####
 # zipkin-ui - An image containing the Zipkin web frontend only, served by NGINX
 #####
 
 FROM nginx:1.18-alpine as zipkin-ui
-LABEL MAINTAINER Zipkin "https://zipkin.io/"
+LABEL MAINTAINER OpenZipkin "http://zipkin.io/"
```

Comment:
just curious about the reason for this change.

Reply:
consistent with other images and project files. I noticed it was different